### PR TITLE
fixes circular dependency issue with aws.py

### PR DESCRIPTION
**Pull request by lsh-0**
aws.find_region has been shifted into utils.
aws.py is probably a deletion candidate

a tiny peace offering in lieu of the gigantic boto3 port coming up.

### CODE REVIEW

File: src/utils.py [hunk 1]
```diff
@@ -118,3 +117,12 @@ def table(rows, keys):
...
     return "\n".join(lines)
+
+def find_region(stackname=None):
+    """tries to find the region, but falls back to user input if there are multiple regions available.
+    Uses stackname, if provided, to filter the available regions"""
+    try:
+        return core.find_region(stackname)
+    except core.MultipleRegionsError as e:
+        print("many possible regions found!")
+        return _pick('region', e.regions())
```
Comment (giorgiosironi) [inline src/utils.py]: feels like the right place to deal with user input

File: src/aws.py [hunk 2]
```diff
@@ -1,35 +1,9 @@
...
-    """tries to find the region, but falls back to user input if there are multiple regions available.
-
-    Uses stackname, if provided, to filter the available regions"""
-    try:
-        return core.find_region(stackname)
-    except MultipleRegionsError as e:
-        print("many possible regions found!")
-        return utils._pick('region', e.regions())
-
-def stack_list(region=None):
```
Comment (giorgiosironi) [inline src/aws.py]: this method was dead code, good to remove it

Comment (lsh-0) [reply src/aws.py]: yeah, always a toss up between marking it as 'dead' or just outright deleting it.

File: src/aws.py [hunk 3]
```diff
@@ -1,35 +1,9 @@
...
-    return core.active_stack_names(region)
-
-#
-#
-#
-
-#
-#
-#
-
```
Comment (giorgiosironi) [inline src/aws.py]: the remaining tasks here do not add much, but seem harmless

**Pull request merged**